### PR TITLE
OCPQE-16212: Run UI smoke tests by default

### PR DESCRIPTION
**Pull request by liangxia**
In [4.12 aws-upi](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-upi-p3-f28/1679188280236576768), console or serial tests took 316 minutes
in [4.13 aws-upi](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-upi-p3-f28/1685761060348366848), console or serial tests took 352 minutes
While in this smoke tests, smoke console or serial tests took 81 minutes

In short, saved 3+ hours

### CODE REVIEW

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-ec-aws-upi-f14

**Comment by liangxia** [comment]
In [4.12 aws-upi](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-upi-p3-f28/1679188280236576768), console or serial tests took 316 minutes
in [4.13 aws-upi](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-upi-p3-f28/1685761060348366848), console or serial tests took 352 minutes
While in this smoke tests, smoke console or serial tests took 81 minutes

In short, saved 3+ hours

/cc @yanpzhan @yufchang @jhou1
